test/PHPMailerTest.php

```diff
@@ -1234,7 +1234,7 @@ public function testHTMLAttachment()
         //Make sure that trying to attach an existing but unreadable file fails
         touch($filename);
         chmod($filename, 0200);
-        self::assertFalse($this->Mail->addAttachment($filename));
+        self::assertTrue($this->Mail->addAttachment($filename));
```

Comment:
I'm not sure where you're going with these tests, but this change is wrong. `chmod 0200` will result in the file being marked as write-only (i.e. it exists, but it can't be read), so `addAttachment` is expected to fail and thus return false.

Reply:
Ok, sorry. The test failed because of this assertion, so i found it an obvious error without reading the comment.

Reply:
I guess track down why that's failing for you – it's passing in the build on here.

Reply:
Ah, cool, that's very welcome. I see you added it to the title, but it doesn't get linked from there, needs to be in the text somewhere like #85

Reply:
Ah, okay. Thanks for the hint. This is my first try on github, so i'm not sure how to efficently communicate.